x-pack/plugins/cross_cluster_replication/public/app/components/follower_index_resume_provider.js

```diff
@@ -78,12 +84,21 @@ class Provider extends PureComponent {
           }
           onMouseOver={this.onMouseOverModal}
         >
-          {!isSingle && (
+          {isSingle ? (
+            <p>
+              <FormattedMessage
+                id="xpack.crossClusterReplication.resumeFollowerIndex.confirmModal.singleResumeDescription"
+                defaultMessage="This follower index will be resumed using default settings. To resume using
+                  different settings, edit this follower index instead."
+              />
+            </p>
+          ) : (
             <Fragment>
               <p>
                 <FormattedMessage
                   id="xpack.crossClusterReplication.resumeFollowerIndex.confirmModal.multipleResumeDescription"
-                  defaultMessage="These follower indices will be resumed:"
+                  defaultMessage="These follower indices will be resumed using default settings. To resume using
+                    different settings, edit each follower index instead:"
```

Comment:
If we're offering an alternative to the user in the modal, should we also add a button (perhaps with the empty style) for editing the index? This would require significant changes since we'd have to use regular `EuiModal` instead of `EuiConfirmModal`. Maybe we could just add this to the roadmap as a nice-to-have?

![image](https://user-images.githubusercontent.com/1238659/51695864-eea8e380-1fb8-11e9-80e6-d52595a4481a.png)

Reply:
I will link `edit this follower index` to the follower form instead, so that we don't have to switch modal components 😄

Reply:
^ this will be done in a followup PR to address more design feedback for this area